express/Question7

```diff
@@ -0,0 +1,42 @@
+ANSWER
+fileutils.js
+// requires
+	var fs = require('fs');                            *** require file system
+
+
+	module.exports = {                                 *** function module readfile asyncronous
+	    readFile: function(file, callback){
+	        fs.readFile(file, function(err, data){
+	            if(err)
+	                callback(err);
+	            else
+	                callback(null, data);
+	        });
+	    }
+	}
+
+index.js
+// requires
+	var express = require('express');                      *** require express ok
+	var fileUtils = require('./fileUtils');                *** require in /fileUtils ok
+
+
+	// initialize
+	var PORT = process.argv[2];                            *** init port ok
+	var FILENAME = process.argv[3] || 'sample.txt';        *** define the file name in use, in this case sample.txt to be located in /books
+	var app = express();                                   *** define app as express server ok
+
+
+	// lets go
+	app.all('/books', function(req, res){                  *** app.all ? '/books' means located in /books directory, the rest of line ok
```

Comment:
1. http://expressjs.com/en/api.html#app.all
1. `/books` is not a directory but merely matches the request path.